docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+services:
+    django:
+        build: ./edagames-django
+        ports:
+            - "8000:8000"
+        volumes: 
+            - ${PWD}/edagames-django/db.sqlite3:/edagames-django/db.sqlite3
+        environment:
+            - SECRET_KEY
+            - SECRET_KEY_JWT
+            - SOCIAL_AUTH_FACEBOOK_KEY
+            - SOCIAL_AUTH_FACEBOOK_SECRET
+            - SERVER_URL=http://server
```

Comment:
Don't need http://server instead could be just server

Reply:
Yes, we tried that... but when we try to make a challenge we discovered that the webpage cannot find/connect with the server because the protocol is missing. 
```
bots_json = requests.get(
            '{}:{}/users'.format(SERVER_URL, SERVER_PORT),
        )
```
To user _server_ we should specify protocol in the string:
```
bots_json = requests.get(
            'http://{}:{}/users'.format(SERVER_URL, SERVER_PORT),
        )
```